pkgs/development/python-modules/hatch-vcs/default.nix

```diff
@@ -38,6 +37,10 @@ buildPythonPackage rec {
     # incompatible with setuptools-scm>=7
     # https://github.com/ofek/hatch-vcs/issues/8
     "test_write"
+    # https://github.com/pypa/setuptools_scm/issues/1038, fixed in setuptools_scm@8.1.0
+    "test_basic"
+    "test_root"
+    "test_metadata"
```

Comment:
If they only break 3.10 and older, then they shouldn't be disabled for all versions.

Reply:
Fixed!